pkg/crc/machine/delete.go

```diff
@@ -20,5 +25,61 @@ func (client *client) Delete() error {
 	if err := libMachineAPIClient.Remove(client.name); err != nil {
 		return errors.Wrap(err, "Cannot remove machine")
 	}
+
+	if err := cleanKubeconfig(getGlobalKubeConfigPath(), getGlobalKubeConfigPath()); err != nil {
+		logging.Warn(err)
+	}
 	return nil
 }
+
+func cleanKubeconfig(input, output string) error {
```

Comment:
I would move this to `kubeconfig.go` file where we have write logic.

Reply:
done